lib/jekyll-github-metadata/repository.rb

```diff
@@ -2,6 +2,33 @@ module Jekyll
   module GitHubMetadata
     class Repository
       attr_reader :nwo, :owner, :name
+
+      # Defines an instance method that delegates to a hash's key
+      #
+      # hash   - a symbol representing the instance method to delegate to. The
+      #          instance method should return a hash or respond to #[]
+      # key    - the key to call within the hash
+      # method - (optional) the instance method the key should be aliased to.
+      #          If not specified, defaults to the hash key
+      #
+      # Returns a symbol representing the instance method
+      def self.def_hash_delegator(hash, key, method = nil)
```

Comment:
The order of these is hard to read. I think this is easier to read:

```
def self.def_hash_delegator(method, hash, key)
```

Having the method name only some of the times is hard to grok. 

I guess matching it with `def_delegator` is a noble goal. In that case, I'd say we should *always* require the `method` argument. Also, for 👀 sake, maybe align each argument in a table-like format for easy scanning?